core/src/main/java/com/linecorp/armeria/common/AbstractHttpRequestBuilder.java

```diff
@@ -217,7 +217,7 @@ public AbstractHttpRequestBuilder queryParam(String name, Object value) {
         if (queryParams == null) {
             queryParams = QueryParams.builder();
         }
-        queryParams.setObject(name, value);
+        queryParams.addObject(name, value);
```

Comment:
Should we update the Javadoc as well? We use `Adds ...` if something is appended.

Reply:
Thanks for the guide. I found where to update it. But it's time to go to work. I will do that tonight.

Reply:
I made a draft. I intended to keep the existing concept as much as possible, so I looked at the HttpRequestSetters.header() javadoc description, which has a similar pattern to queryParam().

But I think it will be helpful for users to understand the method behavior if we slightly modify the example as below.
How about you? 

```
Adds a query param for this request. For example:

HttpRequest.builder()
            .get("/endpoint")
            .queryParam("param", "foo")
            .queryParam("param", "bar")
            .build(); // GET `/endpoint?param=foo&param=bar`
```

Reply:
>  if we slightly modify the example as below

Sounds good to me 👍